### ISSUE TITLE
FileUpload Field Issue within Tabs?

### ISSUE THREAD

**Issue by ryanscherler** [laravel-filament/filament#165]
Seems there is an issue with using tabs and the FileUpload field. Causes runaway network calls (see network console screenshot).

**To reproduce**

Define the follow schema:

```php
    public static function form(Form $form)
    {
        return $form
            ->schema([
                Components\Tabs::make('Page')
                    ->tabs([
                        Components\Tab::make(
                            'SEO',
                            [
                                Components\FileUpload::make('record.seo_image')
                                    ->image()
                                    ->maxSize(5000)
                            ]
                        ),
                    ]),
            ]);
    }
```

Watch the console.... it will be calling the Livewire page resource causing your fan to go wild!

**Context**
- Package version: develop
- Browser: All
- Server OS: [e.g. Mac BigSur 11.2.3]
- PHP version: [e.g. 7.4]
- Database: [e.g. MySQL 8.0]

**Additional details**
Add any other details about the problem here.

**Comment by ryangjchandler** [comment]
@ryanscherler I'm going to try and reproduce this one now so we can get a patch out for it.

**Comment by danharrin** [comment]
Closed by #179.
